include/WCSimPMTObject.hh

```diff
@@ -47,6 +47,8 @@ public:
   G4float  GetmaxQE();
   float    HitTimeSmearing(float);
   G4double GetPMTGlassThickness();
+protected:
+  G4float* GetCollectionEfficiencyArray();
```

Comment:
Remove lines 50 and 51. We should aim to make minimal changes and there is no need to add GetCollectionEfficiencyArray(); to the 20" PMTs in this pull request.